pkg/apiclient/error-translating-cron-workflow-service-client.go

```diff
@@ -63,3 +63,19 @@ func (c *errorTranslatingCronWorkflowServiceClient) DeleteCronWorkflow(ctx conte
 	}
 	return workflow, err
 }
+
+func (c *errorTranslatingCronWorkflowServiceClient) ResumeCronWorkflow(ctx context.Context, req *cronworkflowpkg.CronWorkflowResumeRequest, _ ...grpc.CallOption) (*v1alpha1.CronWorkflow, error) {
+	workflow, err := c.delegate.ResumeCronWorkflow(ctx, req)
+	if err != nil {
+		return nil, grpcutil.TranslateError(err)
```

Comment:
what about updating `TranslateError` to return nil if `err` is nil? You could then just do this:

```golang
return workflow, grpcutil.TranslateError(err)
```

Reply:
Separate PR?

Reply:
Opened #4465